supported-platforms.md

```diff
@@ -0,0 +1,20 @@
+# Supported Platforms
+
+A Kubernetes release consists of a set of Go binaries for various Operating Systems and CPU architectures. Platforms are identified using the Go `<GOOS>-<GOARCH>` parlance.
+
+## Released Binaries
```

Comment:
I think we can map this 1:1 to the container images, right? I guess we just have to mention this fact here, too.

Reply:
Yep that sounds good :+1:

Reply:
@saschagrunert in thinking about this more, the container images we publish don't really map 1:1 here, so I think it makes sense to maybe follow-up with some additional documentation on that